lib/manager/components/ProjectFeedListToolbar.js

```diff
@@ -368,12 +429,29 @@ export default class ProjectFeedListToolbar extends PureComponent<Props> {
                   value={filterOption}
                 >
                   {this.messages(`filter.${filterOption}`)}{' '}
-                  <Badge
-                    style={{backgroundColor: '#babec0'}}>
+                  <Badge style={{ backgroundColor: '#babec0' }}>
                     {filterCounts[filterOption]}
                   </Badge>
                 </OptionButton>
               ))}
+              {project.labels.length > 0 && (
+                <DropdownButton
+                  id='project-feedsource-label-filter-button'
+                  // to match the buttons with badges
+                  style={{ paddingBottom: 7 }}
```

Comment:
This doesn't quite align for me. Perhaps also it would be 

![image](https://user-images.githubusercontent.com/2370911/127191809-1eeef82a-652b-47c9-995a-febc6dc889c6.png)

Reply:
Looks to be a text rendering issue (since it aligns on my display). Have changed the unit to `pt` in hopes that it will work a little better.

Can you let me know if this fixes it for you @landonreed?